api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java

```diff
@@ -257,9 +259,9 @@ public void pauseRun(PipelineRun run) {
             kubernetesManager.deletePod(run.getPodId());
             cloudFacade.stopInstance(instance.getCloudRegionId(), instance.getNodeId());
             kubernetesManager.deleteNode(instance.getNodeName());
-
+            removeUtilizationLevelTags(run);
             run.setStatus(TaskStatus.PAUSED);
-            runManager.updatePipelineStatus(run);
+            runManager.updateRunInfo(run);
```

Comment:
Could you please move `removeUtilizationLevelTags(run)` and `runManager.updateRunInfo(run)` calls to line 241 (just after `runManager.updateServiceUrl`).  Replace calls to `updatePodIP` and `updateServiceUrl` with new `updateRunInfo` method.

Reply:
Done